iceoryx_hoofs/include/iceoryx_hoofs/internal/cxx/helplets.inl

```diff
@@ -155,8 +156,8 @@ inline bool doesEndWithPathSeparator(const string<StringCapacity>& name) noexcep
     {
         return false;
     }
-
-    char lastCharacter = name[name.size() - 1U];
+    // AXIVION Next Construct AutosarC++19_03-A3.9.1: Not used as an integer but as actual character
+    char lastCharacter{name[name.size() - 1U]};
```

Comment:
Please make this `const`.

Reply:
@FerdinandSpitzschnueffler Same as last.

Reply:
see comment above